spec/services/aapb/batch_ingest/pbcore_xml_mapper_spec.rb

```diff
@@ -29,7 +29,7 @@
       end
     end
 
-    it 'maps PBCore XML values to the correct attributes for AssetActor' do
+    it 'maps PBCore XML values to the correct attributes for AssetActor', :focus do
```

Comment:
does this `:focus` need to be removed?

Reply:
Ha ha! Yes, thanks! `:focus` mans that only that test ran in the Travis build probably.